### ISSUE TITLE
MS Word: Stop NVDA from saying the word "style" before it says the style name.

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2952]
**Reported by jhomme on 2013-01-30 12:30**
It takes a bit of extra time to listen to NVDA say the word "style" every time a I navigate onto a new style  in MS Word. I would like NVDA to be able to stop saying the word "style" when I navigat in Word. Perhaps the ability to say it or not should be a toggle.

**Comment by bhavyashah** [comment]
@jcsteh Keeping in mind the original author of this ticket is now out-of-reach, does his https://github.com/nvaccess/nvda/issues/2952#issuecomment-155304236 satisfy your initial concerns? At the least, I agree that introducing a toggle for the discussed behaviour seems unneeded.

**Comment by jcsteh** [comment]
I still think there is a risk of ambiguity here, especially if a lot of different formatting information is being reported. Closing as won't fix.
